docker/build/run

```diff
@@ -29,12 +29,29 @@ except subprocess.CalledProcessError:
     del_group = False
 
 try:
-    subprocess.check_call(['useradd', '--groups=docker,sudo', '--shell=/bin/bash', '--uid=' + os.environ['USER_ID'], '--gid=' + os.environ['GROUP_ID'], os.environ['USER_NAME']])
+    subprocess.check_call([
+        'useradd',
+        '--groups=docker,sudo',
```

Comment:
I understand that you need to use docker in a docker ?

Reply:
You want to say ?
 
> I don't understand why you need to use docker in a docker ?

If yes: for create some containers from a build container.